core/arch/aarch64/codec.h

```diff
@@ -30,8 +30,8 @@
  * DAMAGE.
  */
 
-#ifndef CODEC_H
-#define CODEC_H 1
+#ifndef CODEC_H_
```

Comment:
I seem to remember that identifiers that start with an underscore followed by an uppercase letter are reserved for the implementation so I'm not sure that's a good convention. I think most projects would use "CODEC_H" with neither leading nor trailing underscore.